Add default handler for Single Page Applications (#47)
cloudflare/kv-asset-handler: README.md

  } else return fetch(request)
}
```

### `serveSinglePageApp`

> `serveSinglePageApp` is a custom handler for mapping all "html" requests to the root index.html. For use with Single Page Applications that include in-app routing, such as React Router, VueJS, etc.

Reviewer: ```suggestion
`serveSinglePageApp` is a custom handler for mapping requests to a single root: `index.html`. The most common use case is single-page applications - frameworks with in-app routing - such as React Router, VueJS, etc.
```

#### Example

> This example checks the incoming request to see if it evaluates to an html asset, and if so returns the root index.html, otherwise returns the requested asset (e.g. image, css file, js, etc).

Reviewer: ```suggestion
Check the incoming request to see if it evaluates to an html asset, and if so returns the root index.html; otherwise returns the requested asset (e.g. image, css file, js, etc).
```

```js
import { getAssetFromKV, serveSinglePageApp } from '@cloudflare/kv-asset-handler'

addEventListener('fetch', event => {
  event.respondWith(handleRequest(event.request))
})

async function handleRequest(request) {

>   if (request.url.includes('/docs')) {

Reviewer: Do we want to include the prefix example in this example? I think it might further confuse things

Author: TBH i just copy-pasted, but i'm open to removing this.
